src/reference/config/user-platform-warning.rst

```diff
@@ -0,0 +1,14 @@
+.. warning::
+
+   It is anticipated that most platform settings should be set by
+   site administrators - to see available platforms::
```

Comment:
I'm not so sure about this. Many users install Cylc themselves so don't have any site-config so must do this themselves.

Reply:
And these people will know who they are. What I don't want is people like Tim from 4 years ago creating sets of platforms.